src/ORM/Query.php

```diff
@@ -1299,6 +1299,38 @@ public function insert(array $columns, array $types = [])
         return parent::insert($columns, $types);
     }
 
+    /**
+     * Insert multiple rows of data in one call make use of bulk inserts.
+     *
+     * @param  array  $columns The columns to insert as ['columnName', 'columnName']
+     * @param  array  $values The values to insert as [['columnName' => 'value'], ... ] or ['value', ...]
+     * @param  array  $types A map between columns & their datatypes.
+     * @return $this
+     */
+    public function insertAll(array $columns, array $values, array $types = [])
+    {
+        $query = $this->insert($columns, $types);
```

Comment:
`insert()` returns `$this` so there's no need to assign the retun value. You can just use `$this` instead of `$query` later in the method.